src/container.c

```diff
@@ -465,7 +465,7 @@ static int hyper_rescan_scsi(void)
 	num = scandir("/sys/class/scsi_host/", &list, NULL, NULL);
 	if (num < 0) {
 		perror("scan /sys/class/scsi_host/ failed");
-		return -1;
+		return -ENOENT;
```

Comment:
we can just remove the return value of hyper_rescan_scsi(). @gao-feng  @gnawux WDYT?